app/assets/stylesheets/global.sass

```diff
@@ -379,3 +383,7 @@ h1.header, h2.header
   h5
     font-style: italic
     margin-bottom: 0
+
+// Navbar-collapse
+#sign-in-as, #activities-body
+  z-index: -1
```

Comment:
the z-index set to -1 means that the sign-in-as and the activities body cannot be interacted with at all. 
Maybe you could try to remove it and instead add a `z-index` with a high value (e.g. 100) to `#menu`. Then it should work seamlessly.